### DISCUSSION TITLE
Alternative installation

### DISCUSSION THREAD

**Discussion by ProfLeao** [ReactionMechanismGenerator/RMG-Py#2119] [General]
Is there an initiative to adapt the RMG -Py installation process for PyPi as well and not just via Anacoda as shown on the website? Are there any known difficulties for this type of installation? If not, can I perform the setup for making it available in that repository as well?

**Comment by JacksonBurns** [comment]
Hi @ProfLeao 

Unfortunately, RMG cannot be distributed on PyPI at the time being because many of the dependencies are not themselves Python projects. Symmetry, for example, is a single c file that we compile and distribute on the rmg conda channel, which would not be possible to package on PyPI as-is. For us to move to PyPI, this and many other of our dependencies would need to be made into Python projects.

The good news, though, is that we have also added a new installation method - Docker. [Check out the updated install instructions here](https://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html). While this isn't PyPI, we hope it can alleviate some installation woes.
